lib/datadog/core/configuration/option_definition.rb

```diff
@@ -74,6 +82,14 @@ def depends_on(*values)
             @depends_on = values.flatten
           end
 
+          def env_var(value)
```

Comment:
`<nitpick level=420% bikeshed=yes>`

I feel like `environment` (mimicking Python's `os.environment`) or `env` (mimicking Ruby's `ENV`) look cleaner in the DSL?

Examples:

```
option :foo_bar do |o|
  o.type :string
  o.environment 'DD_FOO_BAR'
end
```

```
option :foo_bar do |o|
  o.type :string
  o.env 'DD_FOO_BAR'
end
```

`</nitpick>`

Reply:
One more vote for `env`.

Reply:
I like `env_var` but I will defer to the team. It's not something I feel particularly strongly about.